common/spacing-borders/spacing-borders-en.html

```diff
@@ -0,0 +1,320 @@
+---
+{
+"title": "Spacing and Borders",
+"language":"en",
+"altLangPage":"spacing-borders-fr.html",
+"breadcrumbs":
+[
+{
+"title": "GCWeb home",
+"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+}
+],
+"secondlevel": false,
+"dateModified": "2024-03-05",
+"share": "true"
+}
+---
+<h2><code>.margin-bottom-none</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm ">
+		<div class="margin-bottom-none">No bottom Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm ">
+		<div class="margin-bottom-small margin-bottom-none">20 pixel bottom Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-bottom-none"&gt;&lt;/div&gt;
+&lt;div class="margin-bottom-small margin-bottom-none"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.margin-bottom-small</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="margin-bottom-small">Small bottom Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>Default bottom Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-bottom-small"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.margin-top-large</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="margin-top-large">Large top Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>Default top Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-top-large"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.margin-top-medium</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="margin-top-medium">Medium top Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>Default top Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-top-medium"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
```

Comment:
Can you move those example at the bottom of the page as they are not recommended. They were only added to ease a transition between WET 3 to WET 4.

Please put it in a section named "Spacing utility not recommended" followed by an expand/collapse to view those item where you will downgrade the h2 for an h3. Something like:

```
<h2>Spacing utilities that are not recommended</h2>
<details>
<summary>Only to ease fast content transition from WET 3 to WET 4</summary>

<h3><code>.margin-bottom-none</code></h3>
....

</details>
```